<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1172</width>
    <height>791</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Customer</string>
  </property>
  <property name="unifiedTitleAndToolBarOnMac">
   <bool>false</bool>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTreeWidget" name="treeWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>191</width>
      <height>741</height>
     </rect>
    </property>
    <column>
     <property name="text">
      <string>商品预订系统</string>
     </property>
    </column>
    <item>
     <property name="text">
      <string>商品目录</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>购物车</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>我的订单</string>
     </property>
     <item>
      <property name="text">
       <string>已付款</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>已发货</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>已签收</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>退货申请</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>退货中</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>已退货</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>已取消</string>
      </property>
     </item>
    </item>
   </widget>
   <widget class="QStackedWidget" name="stackedWidget">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>10</y>
      <width>951</width>
      <height>731</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>2</number>
    </property>
    <widget class="QWidget" name="page">
     <widget class="QTableWidget" name="good_table">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>120</y>
        <width>911</width>
        <height>551</height>
       </rect>
      </property>
      <attribute name="horizontalHeaderDefaultSectionSize">
       <number>290</number>
      </attribute>
      <attribute name="horizontalHeaderStretchLastSection">
       <bool>true</bool>
      </attribute>
      <column>
       <property name="text">
        <string>商品名称</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>价格</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>购买</string>
       </property>
      </column>
     </widget>
     <widget class="QLineEdit" name="lineEdit_good">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>60</y>
        <width>191</width>
        <height>41</height>
       </rect>
      </property>
      <property name="placeholderText">
       <string>商品名称</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_good">
      <property name="geometry">
       <rect>
        <x>250</x>
        <y>60</y>
        <width>111</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>搜索</string>
      </property>
     </widget>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>20</y>
        <width>101</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>搜索商品名称</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="page_2">
     <widget class="QTableWidget" name="shop_table">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>40</y>
        <width>911</width>
        <height>551</height>
       </rect>
      </property>
      <attribute name="horizontalHeaderDefaultSectionSize">
       <number>205</number>
      </attribute>
      <attribute name="horizontalHeaderStretchLastSection">
       <bool>true</bool>
      </attribute>
      <column>
       <property name="text">
        <string>商品</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>数量</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>价格</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>删除</string>
       </property>
      </column>
     </widget>
     <widget class="QPushButton" name="pushButton_pay">
      <property name="geometry">
       <rect>
        <x>790</x>
        <y>610</y>
        <width>111</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>去支付</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_cost">
      <property name="geometry">
       <rect>
        <x>670</x>
        <y>620</y>
        <width>81</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Baskerville Old Face</family>
        <pointsize>15</pointsize>
        <weight>50</weight>
        <bold>false</bold>
       </font>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="textFormat">
       <enum>Qt::AutoText</enum>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QLabel" name="label_cost_2">
      <property name="geometry">
       <rect>
        <x>540</x>
        <y>620</y>
        <width>131</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Baskerville Old Face</family>
        <pointsize>15</pointsize>
        <weight>50</weight>
        <bold>false</bold>
       </font>
      </property>
      <property name="text">
       <string>总计：</string>
      </property>
      <property name="textFormat">
       <enum>Qt::AutoText</enum>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="page_3">
     <widget class="QTableWidget" name="reserve_table">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>120</y>
        <width>911</width>
        <height>551</height>
       </rect>
      </property>
      <attribute name="horizontalHeaderDefaultSectionSize">
       <number>110</number>
      </attribute>
      <attribute name="horizontalHeaderStretchLastSection">
       <bool>true</bool>
      </attribute>
      <column>
       <property name="text">
        <string>订单ID</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>商品数量</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>金额</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>日期</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>联系电话</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>地址</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>状态</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>详情</string>
       </property>
      </column>
     </widget>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>680</x>
        <y>10</y>
        <width>71</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Arial</family>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="text">
       <string>账户：</string>
      </property>
     </widget>
     <widget class="QLabel" name="id_label">
      <property name="geometry">
       <rect>
        <x>750</x>
        <y>10</y>
        <width>131</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Arial</family>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="text">
       <string>XXXXXXXXXX</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_myr">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>60</y>
        <width>191</width>
        <height>41</height>
       </rect>
      </property>
      <property name="placeholderText">
       <string>订单ID</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_myr">
      <property name="geometry">
       <rect>
        <x>250</x>
        <y>60</y>
        <width>111</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>搜索</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_3">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>20</y>
        <width>101</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>搜索订单</string>
      </property>
     </widget>
     <widget class="QToolButton" name="toolButton">
      <property name="geometry">
       <rect>
        <x>890</x>
        <y>20</y>
        <width>47</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>...</string>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1172</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
